### PR TITLE
Extra instructions for removing builds from coreos-pool

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/rebase.md [hunk 1]
```diff
@@ -12,21 +12,36 @@
...
+rpm -qai | grep -B 8 $key
+```
+
+If there are any RPMs signed by the old key they'll need to be investigated. Maybe they shouldn't be used any longer. Or maybe they're still needed.
+
+After verifying the list looks good:
+
+```
+koji untag-build coreos-pool $untaglist
 ```
```
Comment (jlebon) [inline .github/ISSUE_TEMPLATE/rebase.md:45]: WDYT about adding something here like:

> Give a heads up to rpm-ostree developers that N-2 packages have been untagged and that they may need to update their CI compose tests to freeze on a newer FCOS commit.

?

**Review by jlebon — APPROVED** [review]
One optional comment, LGTM otherwise!

File: .github/ISSUE_TEMPLATE/rebase.md (hunk 1 above)
Comment (dustymabe) [reply .github/ISSUE_TEMPLATE/rebase.md:45]: done

**Review by jlebon — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/rebase.md [hunk 2]
```diff
@@ -12,22 +12,39 @@
 - `koji taginfo coreos-pool`
 - `koji edit-tag coreos-pool -x tag2distrepo.keys="12c944d0 9570ff31 45719a39"`
 
-- [ ] `koji untag` N-2 packages from the pool (at some point we'll have GC in place to do this for us, but for now we must remember to do this manually or otherwise distRepo will fail once the signed packages are GC'ed). For example the following snippet finds all RPMs signed by the Fedora 31 key and untags them.
+- [ ] `koji untag` N-2 packages from the pool (at some point we'll have GC in place to do this for us, but for now we must remember to do this manually or otherwise distRepo will fail once the signed packages are GC'ed). For example the following snippet finds all RPMs signed by the Fedora 32 key and untags them.
+
+Find the key short hash. Usually found [here](https://pagure.io/fedora-infra/ansible/blob/main/f/roles/bodhi2/backend/templates/pungi.rpm.conf.j2). Then:
```
[the remark this answers is not in the document]
Comment (bgilbert) [reply .github/ISSUE_TEMPLATE/rebase.md:17]: https://github.com/coreos/fedora-coreos-tracker/pull/810

**Pull request merged**